### ISSUE TITLE
appositives in mini-dhbb

### ISSUE THREAD

**Issue by vcvpaiva** [cpdoc/dhbb-nlp#78]
As @arademaker noticed there are meaning differences between
>José Eduardo Siqueira Campos nasceu em Campinas (SP).

as Campinas is not the same as Sao Paulo

and
>José Eduardo estudou no Centro de Ensino Unificado de Brasília (CEUB)

as CEUB is exactly the same as Centro de Ensino Unificado de Brasília.

but the syntax is the same: do we annotate as the same relation or not?

**Comment by arademaker** [comment]
No, I am planning to adopt nmod for thx first and appos for the second.

**Comment by vcvpaiva** [comment]
But Stanza says it's parataxis in the sentence
>José Eduardo Siqueira Campos nasceu em Campinas (SP) no dia 4 de março de 1959.

**Comment by arademaker** [comment]
Stanza is saying what it saw in the Bosque, it doesn't mean it is right. Indeed, parataxis is one possible analysis https://universaldependencies.org/u/dep/parataxis.html but it is the less informative one.
